.travis.yml

```diff
@@ -20,3 +20,4 @@ branches:
   only:
   - master
   - devel
+  - /^\d+\.\d+\.\d+(-SNAPSHOT)?$/ # ensure builds are run on tags which have versions as branch names
```

Comment:
"Trigger builds on tags which are named '<number>.<number>.<number>(-optional)' to ship the SDK."